xml/hardening.xml

```diff
@@ -1704,21 +1704,36 @@ FIXME:
     <title>Firewall (iptables)</title>
 
     <para>
-     iptables will not be covered in detail in this guide. Most companies use
-     dedicated firewalls or appliances to protect their servers in a production
-     network. This is strongly recommended for secure environments.
-     &productname; also includes &susefirewall; which is a wrapper for iptables
-     and is enabled by default as a simple and layered protection.
+     Setting up a Linux firewall from scratch will not be covered in detail in
+     this guide. Most companies use dedicated firewalls or appliances to
+     protect their servers in a production network. This is strongly
+     recommended for secure environments.
     </para>
 
     <para>
-     If you are also interested in Linux stateful firewalls using iptables,
-     there are many guides on the Internet. See the Appendix for resources. For
-     lots of iptables tutorials and examples, see
-     <link xlink:href="http://www.linuxguruz.com/iptables/"/>.
-<!-- An overview is also
-     available at <xref linkend="sec.hardening.firewall.iptables"/>. -->
+     &productname; also includes &firewalld;. This is a
+     firewall daemon that can be configured from the command line using
+     <command>firewall-cmd</command> or in a graphical user interface
+     using <command>firewall-config</command>. Basic settings of &firewalld;
+     can also be configured via the &yast; firewall interface. For detailed
+     information on how &firewalld; works, refer to <link
+     xlink:href="http://www.firewalld.org/documentation/"> the official
```

Comment:
remove "the official documentation" and make link a single tag. We display the full URLs in order to make printouts usable.